src/sql/workbench/services/tableDesigner/common/interface.ts

```diff
@@ -37,5 +37,5 @@ export interface ITableDesignerService {
 	 * @param providerId The provider id
 	 * @param tableInfo The table information
 	 */
-	openTableDesigner(providerId: string, tableInfo: azdata.designers.TableInfo, telemetryInfo?: ITelemetryEventProperties): Promise<void>;
+	openTableDesigner(providerId: string, tableInfo: azdata.designers.TableInfo, telemetryInfo?: ITelemetryEventProperties, objectExplorerContext?: azdata.ObjectExplorerContext): Promise<void>;
```

Comment:
add doc comment (for telemetry info too since that seems to have been missed)

Reply:
done